packages/lit/package.json

```diff
@@ -247,7 +247,9 @@
         "test/*_test.html",
         "development/test/*_test.html",
         "test/polyfill-support/*_test.html",
-        "development/test/polyfill-support/*_test.html"
+        "development/test/polyfill-support/*_test.html",
+        "lit-core.min.js{,.map}",
```

Comment:
Does this mean these will be included in releases? And is this new? If so, is that stressful at all?

Reply:
No, this is just communicating that these files are generated by this command to wireit. Should have no impact beyond wireit caching etc